bb8/src/inner.rs

```diff
@@ -244,3 +271,39 @@ impl<M: ManageConnection> Reaper<M> {
         }
     }
 }
+
+struct SharedPoolInnerStatistics {
+    gets: AtomicU64,
+    gets_waited: AtomicU64,
+}
+
+impl SharedPoolInnerStatistics {
+    fn new() -> Self {
```

Comment:
Let's get rid of `new()` in favor of deriving `Default`.

Reply:
Just for curiosity - kinda new in Rust - is this just a pattern detail or has some implications?

Reply:
It's just less code because all the values here are the `Default` values for their respective types anyway. Less code is almost always better!

Reply:
Good point, thanks for the clarification.